packages/cli/stores/config.ts

```diff
@@ -101,6 +102,20 @@ class ConfigStore {
       const content = await readFile(this.configPath, "utf-8");
       const parsed = parseJSON(content) as unknown as Partial<Config>;
 
+      if (packageJSONPath) {
```

Comment:
will this find the CLI package json or the project package JSON?

Reply:
can we test this out end to end to make sure?

Reply:
Right, good question. I think the later... When testing it worked ofc as `package.json` is the same but bundled it won't be, but I'll take another look. Annoying we can't directly import json files without the experimentation warning like we do in the other SDKs.

Reply:
I have resolved this and tested end-to-end locally with https://github.com/bucketco/bucket-web/pull/2634